app/views/main/_verify_inline_signature.html.erb

```diff
@@ -1,10 +1,16 @@
 <p class="justify">
-  <%=_("Many applications can verify the checksum of a download. To verify your download can be important as it verifies you really have got the ISO file you wanted to download and not some broken version.") %>
-  <%=_("You could verify the file in the process of downloading. For example a checksum (SHA256) will be used automatically if you choose <a href='http://en.opensuse.org/SDB:Metalink'>Metalink</a> in the field above and use the add-on DownThemAll! in <a href='http://en.opensuse.org/Firefox'>Firefox</a>.") %>
+  <%=_("To avoid installation errors or boot failures the integrity of the downloaded file should be checked.") %>
+  <%=_("To have the download tool automatically verify the checksum, choose the <a href='http://en.opensuse.org/SDB:Metalink'>Metalink</a> in the field above and use an add-on like e.g DownThemAll! for <a href='http://en.opensuse.org/Firefox'>Firefox</a>.") %>
 </p>
 <p>
-  <%=_("For each ISO, we offer a checksum file with the corresponding SHA256 sum.") %>
-  <%=_("For extra security, you can use GPG to verify who signed those .sha256 files. It should be %s.") % @gpg %>
+  <%=_("Alternatively, use the sha256sum tool to verify the checksum manually:") %>
+<noscript>
```

Comment:
No sure if this has some other meaning in `.erb`. Looking at the live page and this template there are two copies of links to checksum file, one in `<noscript>` and one in `<p class="hidden script_only">`. The [<noscript>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript) is used to inject html when javascript is disabled.

Seems like the use here you are looking for code formatting like a lot of people use `<code>` or `<pre>`.

Obviously, since I am not familiar with this site I could be way off. Thanks for working on the bug!